docs/team/irfan.md

```diff
@@ -3,44 +3,74 @@ layout: page
+taught during the Software Engineering Module CS2103T.
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added Importing functionality
+  * What it does: Allows users to input contacts CMM's database quickly. CMM currently imports from Excel Files only
+  * Justification: Based on the user story, a telemarketer receives daily call list in an Excel sheet. It required an 
+    in-depth analysis of design alternatives. This function is an integral part in  ensuring a smooth transition
+    between the manager to the telemarketer. This process reduces learning curve, as well as overall time of adding in
+    contacts (previously the two options were to either learn json or to manually input  all the data) 
+  * Highlights: This enhancement affects existing commands and commands to be added in the future. The type of data
+    imported will affect what and how the commands interact with the data found in CMM. This enhancement will also be
+    relevant and often updated as long as CallMeMaybe uses a local storage, and the company  processes  include  Excel
+    sheets  in storing customer data. The implementation was challenging as it required constant updating and tweaking
+    as the team's definition of a valid person  to be imported constantly changed throughout the project
+  * Credits: *The previous iteration CallMeMaybe (AddressBookLevel3) 's JSONstorage helped me structure my ImportExport
+    feature. This taught me how to follow an existing architectural style while implementing a new feature. It also
+    helped me rethink the way I approach code abstractions way before the CS2103T's lecture of code architecture*
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
-
-* **Code contributed**: [RepoSense link]()
+* **New Feature**: Added Exporting functionality
+  * What it does: Allows users to export contacts CMM's database quickly. CMM currently export to Excel Files only
+  * Justification: Based on the user story, a telemarketer returns to the manager the daily call list in an Excel sheet.
```

Comment:
Perhaps change this sentence to 'a telemarketer returns the daily call list in an excel sheet to the manager' to make it more readable? A trivial issue.